src/app/rundown-view/components/unfocused/unfocused.component.ts

```diff
@@ -19,4 +19,12 @@ export class UnfocusedComponent {
   public onFocus(): void {
     this.isUnfocused = false
   }
+
+  public getCenterPositionForLabel(): string {
+    const isWindowHeightLargerThanCurrentScreenHeight: boolean = window.innerHeight > window.screen.availHeight
+    if (!isWindowHeightLargerThanCurrentScreenHeight) {
+      return `${window.innerHeight / 2}`
+    }
+    return `${window.screen.availHeight / 2}`
+  }
```

Comment:
Instead of triggering the change detection so much, let's make it use a variable for the margin :D Using function calls will trigger a lot of change detection :)

```suggestion
@HostListener('window:resize')
  public onResize(): void {
    this.marginTop = this.getCenterPositionForLabel()
  }

  public marginTop = 0

  public getCenterPositionForLabel(): number {
    const isWindowHeightLargerThanCurrentScreenHeight: boolean = window.innerHeight > window.screen.availHeight
    if (!isWindowHeightLargerThanCurrentScreenHeight) {
      return window.innerHeight / 2
    }
    return window.screen.availHeight / 2
  }
```

Reply:
I second this suggestion :)
Only small change: We want to keep our instance variables (`marginTop`) above our functions and constructor.

Reply:
It has been implemented! Thanks for the great suggestion :-)